.travis.yml

```diff
@@ -10,6 +10,7 @@ env:
   - E2E_TEST_HOME=$GOPATH/src/github.com/ava-labs/avalanche-testing/
   - COMMIT=${TRAVIS_COMMIT::8}
   - DOCKERHUB_REPO=avaplatform/avalanchego
+  - BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
```

Comment:
@holisticode @aaronbuchwald 
Can you review this ?

A lot of people are complaining about in https://github.com/travis-ci/travis-ci/issues/6652 ?

Reply:
This looks like the solution suggested in https://unhashable.com/getting-the-current-branch-name-during-a-pull-request-in-travis-ci/. So if the desired branch for a PR is indeed

> the name of the branch from which the PR originated, if the current job is a pull request

then this looks good to me.

However, I doubt this works in a yml file like that and probably requires a shell script.

Reply:
Yeah makes sense all right - let me revert this 👍